Felienne/hedy
# Adding margin between debugging and edit button

**Description**
Added a bit of margin to the debugging and edit button. 

**Fixes issue #2775**

**How to test**
Save & share your code. Go to the link that Hedy provides. Make sure that the is some margin/space between the debug button and the edit button. It should look like:

![image](https://user-images.githubusercontent.com/33595265/169338042-b410f6f8-448f-46a8-a0b6-6829a0fa6af0.png)
